INSTALL.md

```diff
@@ -50,7 +50,7 @@ brew install pkg-config
 ## Mac OS X NodeJS setup
 
 ```
-curl -L https://git.io/n-install | bash -s -- -y lts
+curl -L https://git.io/n-install | bash -s -- -y 8.12
```

Comment:
```  
Supported version specifiers:

  * lts    ... the LTS (long-term stability) version
  * latest ... the latest version available overall
  * otherwise, specify an explicit version number, such as '0.12' or '0.10.35'
```

No way to do this with this tool!